cmd/demo/node_setup.go

```diff
@@ -120,19 +129,19 @@ func (n *node) setupContracts() error {
 			assAddr, err = validateAssetHolder(n.cb, adjAddr) // validate asset holder
 		}
 	case contractSetupOptionDeploy:
-		if adjAddr, err = deployAdjudicator(n.cb); err == nil { // deploy adjudicator
-			assAddr, err = deployAssetHolder(n.cb, adjAddr) // deploy asset holder
+		if adjAddr, err = n.deployAdjudicator(n.cb); err == nil { // deploy adjudicator
```

Comment:
background: i made it a member function because deployment now requires an account and that the node account is stored with the node struct.

will apply your suggestions.

Reply:
I also think its better to not have them as member functions.